Library/Formula/reflex.rb

```diff
@@ -0,0 +1,19 @@
+class Reflex < Formula
+  desc "Variant of the flex fast lexical scanner"
+  homepage "http://invisible-island.net/reflex/reflex.html"
+  url "http://invisible-island.net/datafiles/release/reflex.tar.gz"
```

Comment:
It is stable, and the version is inside the tarball (i.e. when extracted, the version shows in the extracted folder). Unfortunately, that's how the author of the software hosts it and I can't change that.